utils/python/MITgcmutils/MITgcmutils/density.py

```diff
@@ -38,8 +38,8 @@ def _check_salinity(s):
 
     sneg = s<0
     if not np.any(sneg):
-        warnings.warn('found negative salinity values, reset them to NaN')
-        # s[sneg] = np.NaN
+        warnings.warn('found negative salinity values, reset them to nan')
+        # s[sneg] = np.nan
```

Comment:
If we don't reset the values to nan, than we may not want to say so, e.g. like this:
```suggestion
        warnings.warn('found negative salinity values')
        # warnings.warn('found negative salinity values, reset them to nan')
        # s[sneg] = np.nan
```

Reply:
Definitely!  Want to do the honors?